test/test_autograd.py

```diff
@@ -2808,6 +2809,22 @@ def test_profiler_aggregation_lstm(self):
             with tempfile.NamedTemporaryFile() as trace_file:
                 prof.export_chrome_trace(trace_file.name)
 
+    def test_record_function(self):
+        x = torch.randn(10, 10)
+
+        with profile() as p:
+            with record_function("label"):
```

Comment:
What happens if we have multiple levels of context managers? I suppose only the name of the inner-most one appears?

I was wondering if it would make sense to also nest the names of the contexts, so that users can easily understand and parse profiles in arbitrary places of code (like `classifier.block1` instead of just `block1`)

Reply:
Nope, you'll see them as nested ranges in the trace.